setools/policyrep/selinuxpolicy.pxi

```diff
@@ -351,8 +351,11 @@ cdef class SELinuxPolicy:
     #
     def lookup_boolean(self, name):
         """Look up a Boolean."""
+        cdef:
+            char *new_name = selinux.selinux_boolean_sub(name)
```

Comment:
This is a memory leak since the C string array is not freed. Cython won't automatically free it. One option is to convert it to a Python string and immediately free it, but I believe it also needs NULL return handling here.

Reply:
Thank you. Updated.